<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="UTF-8">
    <title>po21 Precision Oncology</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../stylesheets/github-light.css" media="screen">
  </head>
  <body>
    <section class="page-header">
      <h1 class="project-name">po21 Precision Oncology</h1>
      <h2 class="project-tagline">Instituto Gulbenkian de Ciência <br> 25-29 October 2021</h2>
    </section>

    <section class="main-content">
      <h1>
<a id="course-description" class="anchor" href="#course-description" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Course description</h1>

<p>Precision Oncology (PO) is already revolutionizing healthcare and will play a dominant role in the future of cancer therapy. PO integrates tumor multi-omic profiles and data that reflect the course of the disease, lifestyle and environment to guide clinical decisions during cancer patient journey such as prevention, diagnosis and treatment. Bioinformatics analyses are essential to identify patients who will benefit from treatment based on their molecular profile, and to tailor chemotherapeutic regimens accordingly. 
The aim of the course is to present a complete computational pipeline for the analysis and interpretation of Next-Generation Sequencing (NGS) data such as exome sequencing or targeted panels that are commonly used in the clinic. We will address the implementation of large-scale genomic sequencing in clinical practice and the recently developed computational strategies for the analysis of NGS data with a particular emphasis on the interpretation of the results, selection of biomarkers of drug response and afford opportunities to match therapies with the characteristics of the individual patient’s tumor. Exercises and case studies focused on cancer will be used to illustrate the principles of how genetics influence led to refining diagnoses and personalized treatment of cancer disease. Although focused on cancer, some of the principles and steps could be extrapolated to other complex diseases.
</p>

<h1>
<a id="target-audience" class="anchor" href="#target-audience" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Target audience</h1>

<p>This course is intended for working healthcare professionals and Bioinformaticians working in the area.</p>

<h1>
<a id="pre-requisites" class="anchor" href="#pre-requisites" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Pre-requisites</h1>

<p>The course assumes that attendees are not intimidated by the prospect of gaining experience working on UNIX-like operating systems (including the shell, and shell scripting). Attendees should understand some of the science behind high-throughput DNA sequencing and sequence analysis, as we will not go deeply into underlying theory (or the mechanics of given algorithms, for example) as such. What will be taught are technical solutions for automating and sharing such analyses in shareable, reusable compute environments, which will include (but is not limited to) beginner-level programming, and basic Linux provisioning. General computer literacy, (e.g. editing plain text data files, navigating the command line) will be assumed.</p>

<h1>
<a id="detailed-program" class="anchor" href="#detailed-program" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Detailed program</h1>

<ul>
<li>
<strong>Mon, Oct 25th</strong> Day #1 Introduction and methods <br>
09:30 - 10:00 Introduction to the course and self presentation of the participants <br>
<!--10:00 - 11:00 <a href="https://github.com/cniobu/pm17/raw/gh-pages/material/01_PersonalizedMedicine.pdf">Personalized Medicine</a> <br> -->
10:00 - 11:00 Personalized Medicine <br>
<span style="color:#b3b6b7">11:00 - 11:30 <small>Coffee Break</small></span><br>
<!--11:30 - 12:30 <a href="https://github.com/cniobu/pm17/raw/gh-pages/material/02_Variant_detection.pdf">NGS I: Variant detection</a><br>-->
11:30 - 12:30 Workflow <br>
<span style="color:#b3b6b7">12:30 - 14:00 <small>Lunch Break</small></span><br>
<!--14:00 - 16:00 <a href="https://github.com/cniobu/pm17/raw/gh-pages/material/03_data_n_methods.pdf">Playing with the data and the methods</a><br>-->
14:00 - 15:30 Data formats <br>
<span style="color:#b3b6b7">15:30 - 16:00 <small>Coffee Break</small></span><br>
<!--16:30 - 18:00 1st Exercise: <a href="https://github.com/cniobu/pm17/raw/gh-pages/material/04_running_the_pipelineVp.pdf" style="color:orange">Running the pipeline</a> | [<a href="https://github.com/cniobu/pm17/raw/gh-pages/material/04_running_the_pipeline.pdf" style="color:orange">Solved</a>] | [<a href="https://www.dropbox.com/s/5pxdlq0koo4hom8/OVCA_case.tar?dl=0">OVCA_case data</a>] </li>-->
16:00 - 17:30 Quality control<br>
<!--16:30 - 18:00 Exercise: Running the pipeline </li>-->
<li>
<strong>Tue, Oct 26th</strong> Day #2 Variant Calling <br>
<!-- 09:30 - 10:00 Introduction to the course and self presentation of the participants <br> -->
<!--10:00 - 11:00 <a href="https://github.com/cniobu/pm17/raw/gh-pages/material/01_PersonalizedMedicine.pdf">Personalized Medicine</a> <br> -->
09:30 - 10:00 Review Day 1 <br>
<!--11:30 - 12:30 <a href="https://github.com/cniobu/pm17/raw/gh-pages/material/02_Variant_detection.pdf">NGS I: Variant detection</a><br>-->
10:00 - 11:00 NGS I: Variant detection II<br>
<span style="color:#b3b6b7">11:00 - 11:30 <small>Coffee Break</small></span><br>
11:30 - 12:30 Exercise I <br>
<span style="color:#b3b6b7">12:30 - 14:00 <small>Lunch Break</small></span><br>
14:00 - 15:30 Exercise II - Part 1 <br>
<span style="color:#b3b6b7">15:30 - 16:00 <small>Coffee Break</small></span><br>
<!--14:00 - 16:00 <a href="https://github.com/cniobu/pm17/raw/gh-pages/material/03_data_n_methods.pdf">Playing with the data and the methods</a><br>-->
16:00 - 17:30 Exercise II - Part 2 <br>
<!--16:30 - 18:00 1st Exercise: <a href="https://github.com/cniobu/pm17/raw/gh-pages/material/04_running_the_pipelineVp.pdf" style="color:orange">Running the pipeline</a> | [<a href="https://github.com/cniobu/pm17/raw/gh-pages/material/04_running_the_pipeline.pdf" style="color:orange">Solved</a>] | [<a href="https://www.dropbox.com/s/5pxdlq0koo4hom8/OVCA_case.tar?dl=0">OVCA_case data</a>] </li>-->
16:30 - 18:00 Discussion </li>
<li>
<strong>Wed, Oct 27th</strong> Day #3 Annotation and filtering <br>
09:30 - 10:00 Review day 2 <br>
<!--10:00 - 11:00 <a href="https://github.com/cniobu/pm17/raw/gh-pages/material/05_Variant_annotation.pdf">NGS II: Variant annotation</a> <br> -->
10:00 - 11:00 NGS II: Variant annotation <br>
<span style="color:#b3b6b7">11:00 - 11:30 <small>Coffee Break</small></span><br>
<!--11:30 - 12:30 2nd Exercise: <a href="https://github.com/cniobu/pm17/raw/gh-pages/material/06_Variant_annotation_exercise.pdf" style="color:orange">Running the pipeline</a> <br>-->
11:30 - 12:30 Exercise: Variant annotation <br>
<span style="color:#b3b6b7">12:30 - 14:00 <small>Lunch Break</small></span><br>
<!--14:00 - 16:00 <a href="https://github.com/cniobu/pm17/raw/gh-pages/material/07_Variant_filtering.pdf">Selecting the most relevant variants: How to filter</a> <br>-->
14:00 - 15:30 Selecting the most relevant variants<br>
<span style="color:#b3b6b7">15:30 - 16:00 <small>Coffee Break</small></span><br>
<!--16:30 - 18:00 3rd Exercise: <a href="https://github.com/cniobu/pm17/raw/gh-pages/material/08_Variant_filtering_exercise.pdf" style="color:orange">Playing with the results</a></li>-->
16:00 - 17:30 Exercise: Variant prioritization </li>
<li>
<strong>Thu, Oct 28th</strong> Day #4 Interpretation of the results <br>
09:30 - 10:00 Review day 3 <br>
<!--10:00 - 11:00 <a href="https://github.com/cniobu/pm17/raw/gh-pages/material/09_PanDrugs.pdf">PanDrugs: Matching mutations with therapies</a> <br>-->
10:00 - 11:00 PanDrugs: Matching mutations with therapies <br>
<span style="color:#b3b6b7">11:00 - 11:30 <small>Coffee Break</small></span><br>
<!--11:30 - 12:30 4th Exercise: <a href="https://github.com/cniobu/pm17/raw/gh-pages/material/10_PanDrugs_exercise.pdf" style="color:orange">Running PanDrugs</a> <br>-->
11:30 - 12:30 Exercise: Running PanDrugs <br>
<span style="color:#b3b6b7">12:30 - 14:00 <small>Lunch Break</small></span><br>
<!--14:00 - 16:00 <a href="https://github.com/cniobu/pm17/raw/gh-pages/material/11_CancerGenomicsResources.pdf">Cancer Genomics Resources</a> <br>-->
14:00 - 15:30 Cancer Genomics Resources <br>
<span style="color:#b3b6b7">15:30 - 16:00 <small>Coffee Break</small></span><br>
16:00 - 17:30 Exercise: cBioPortal</li>
<li>
<strong>Fri, Oct 29th</strong> Day #5 Case studies in Personalized cancer medicine <br>
09:30 - 10:00 Review day 4 <br>
10:00 - 11:00 Analysis I <br>
<span style="color:#b3b6b7">11:00 - 11:30 <small>Coffee Break</small></span><br>
11:30 - 12:30 Analysis II <br>
<span style="color:#b3b6b7">12:30 - 14:00 <small>Lunch Break</small></span><br>
14:00 - 16:00 Interpretation &amp; Discussion <br>
<!--  &nbsp;&nbsp;&nbsp; <a href="https://github.com/cniobu/pm17/raw/gh-pages/material/GuidelinesVariantDetection.pdf" >Guidelines for the interpretation of somatic mutations</a> <br>
<span style="color:#b3b6b7">16:00 - 16:30 <small>Coffee Break</small></span><br>-->
16:30 - 17:30 Final Wrap-up session</li>
<!--<a href="https://github.com/cniobu/pm17/raw/gh-pages/material/Software_installation.pdf">Software Installation</a><br>-->
<!--<a href="https://github.com/cniobu/pm17/raw/gh-pages/material/PM17_SoftDep.pdf">Software Installation Quick Guide (root user)</a><br>-->
</ul>

<h1>
<!--<a id="gl0ossary" class="anchor" href="#glossary" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Glossary</h1>-->
<!--<h2 class="project-tagline"><a href="https://github.com/cniobu/pm17/raw/gh-pages/material/Glossary.pdf" style="color:green">File with terms</a></h2>-->

<h1>
<a id="resources" class="anchor" href="#resources" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Resources</h1>
<li>
  Gómez-López G, Dopazo J, Cigudosa JC, Valencia A, <u>Al-Shahrour F</u>. <a href="https://www.ncbi.nlm.nih.gov/pubmed/29077790">Precision medicine needs pioneering clinical bioinformaticians.</a> Brief Bioinform. 2017 Oct 25 <br>
<li>
Rehm HL. <a href="https://www.ncbi.nlm.nih.gov/pubmed/28138143">Evolving health care through personal genomics.</a>Nat Rev Genet. 2017 Apr;18(4):259-267<br>
<h2><a id="clinical-interpret-var" class="anchor" href="#clinical-interpret-var" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Clinical Interpretation of variants</h2>
<ul>
<li> Li MM <i>et al</i>. <a href="https://www.ncbi.nlm.nih.gov/pubmed/27993330">Standards and Guidelines for the Interpretation and Reporting of Sequence Variants in Cancer: A Joint Consensus Recommendation of the Association for Molecular Pathology, American Society of Clinical Oncology, and College of American Pathologists.</a>J Mol Diagn. 2017 Jan;19(1):4-23<br>
<li>
  Tsang H, Addepalli K, Davis SR. <a href="https://www.ncbi.nlm.nih.gov/pubmed/ 28975082">Resources for Interpreting Variants in Precision Genomic Oncology Applications.</a> Front Oncol. 2017 Sep 19;7:214<br>
<li>
  Meric-Bernstam F <i>et al. </i><a href="http://www.ncbi.nlm.nih.gov/pubmed/25863335">A decision support framework for genomically informed investigational cancer therapy.</a> J Natl Cancer Inst. 2015 Apr 11;107(7)<br>
<li>
  Amendola LM <i>et al. </i><a href="http://www.ncbi.nlm.nih.gov/pubmed/27392081">Performance of ACMG-AMP Variant-Interpretation Guidelines among Nine Laboratories in the Clinical Sequencing Exploratory Research Consortium.</a> Am J Hum Genet. 2016 Jul 7;99(1):247<br>
<li>
  Dienstmann R <i>et al. </i><a href="http://www.ncbi.nlm.nih.gov/pubmed/24768039">Standardized decision support in next generation sequencing reports of somatic cancer variants.</a> Mol Oncol. 2014 Jul;8(5):859-73<br>
<li>
  Sukhai MA <i>et al. </i><a href="http://www.ncbi.nlm.nih.gov/pubmed/25880439">A classification system for clinical relevance of somatic variants identified in molecular profiling of cancer.</a> Genet Med. 2016 Feb;18(2):128-36<br>
</ul>

<h2>
<a id="authors-and-contributors" class="anchor" href="#authors-and-contributors" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Authors and Contributors</h2>
<p>Coral Fustero-Torre, María José Jiménez-Santos, Elena Piñeiro-Yáñez, Fátima Al-Shahrour & Pedro L. Fernandes</p>
<!--<p><i>Thanks to Miriam Rubio-Camarillo, José María Fernandez-González & Gonzalo Gómez for the support and development of RUbioSeq.</i></p>-->
      <footer class="site-footer">

        <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
      </footer>

    </section>

  
  </body>
</html>
